styles/style.css

```diff
@@ -0,0 +1,51 @@
+* {
+    box-sizing: border-box;
+}
+body {
+    font-family: 'Roboto', sans-serif;
+    color: #383838;
+    margin: 0;
+}
+.offer-container {
+    max-width: 1440px;
+    height: 569px;
+    background-color: #ffff;
+    padding: 120px 104px 122px;
+}
+.offer-title {
+    margin-bottom: 20px;
+    margin-top: 0;
+    text-align: center;
+    font-weight: 500;
+    font-size: 40px;
+}
+.offer-title-line {
+    width: 100px;
+    height: 4px;
+    background-color: #F5BD41;
+    margin: 0 auto 80px;
+}
+.block {
+    display: inline-block;
+    float: left;
```

Comment:
this property good only for image inside text. you don't need this here, this will broke your next section without [clearfix](https://www.w3schools.com/howto/howto_css_clearfix.asp)

Reply:
Replaced float for overflow: auto.